app/src/main/kotlin/cz/covid19cz/erouska/ui/mydata/MyDataFragment.kt

```diff
@@ -31,6 +31,10 @@ class MyDataFragment :
         super.onViewCreated(view, savedInstanceState)
         enableUpInToolbar(false)
         setupTabs()
+
+        data_collection_info.setOnClickListener {
+            navigate(R.id.action_nav_my_data_to_nav_help, Bundle().apply { putString("type", "data_collection") })
```

Comment:
Why not to use SafeArgs plugin?

Reply:
we didn't use it anywhere in the project yet
but no problem, I can add it